### PR TITLE
fix side bar style for various screen orientations

### PR DESCRIPTION
**Pull request by NoahFisher**
Fixes https://github.com/vim-awesome/vim-awesome/issues/42

vim_awesome_and_vim_awesome

vim_awesome

### CODE REVIEW

File: web/static/sass/screen.scss [hunk 1]
```diff
@@ -295,8 +302,16 @@ code {
...
 
     @media #{$large-screen} {
-      position: absolute;
       bottom: 0;
+      position: absolute;
+    }
+
+    @media #{$not-tall-screen},
+           #{$not-large-screen},
+           #{$large-screen} and #{$not-tall-screen} {
```
Comment (captbaritone) [inline web/static/sass/screen.scss]: Why is this applied for large screens? Should't it just be for "not tall" screens?

File: web/static/sass/screen.scss [hunk 2]
```diff
@@ -295,8 +302,16 @@ code {
...
+
+      margin: 0;
```
Comment (captbaritone) [inline web/static/sass/screen.scss]: What if we gave it some top margin so that it stayed separated from the other items?

**Comment by NoahFisher** [comment]
@captbaritone updated!

**Comment by captbaritone** [comment]
Thanks!

**Comment by captbaritone** [comment]
Deployed.

**Pull request merged**